spotty/spotty/Scenes/Detail/DetailItem/DetailItemViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  DetailItemViewController.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 24.06.22.
+//
+
+import UIKit
+
+class DetailItemViewController: UIViewController {
+    
+    private var imageView: UIImageView!
+    private var titleLabel: UILabel!
+    private var descriptionTextView: UITextView!
+    private var dismissButton: UIButton!
+    
+    private var viewModel: DetailItemViewModelProtocol!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
+        setupUI()
+        bind()
+    }
+    
+    override var prefersStatusBarHidden: Bool { true }
+    
+    private func setupUI() {
+        createImageView()
+        createText()
+    }
+    
+    private func createImageView() {
+        imageView = UIImageView()
+        view.addSubview(imageView, anchors: [.top(0), .leading(0), .trailing(0), .height(view.bounds.width)])
+        createDismissButton()
+    }
+    
+    private func createDismissButton() {
+        let button = UIButton()
+        button.setCustomStyle(.dismiss)
+        view.addSubview(button, anchors: [.top(20), .trailing(-20), .height(35), .width(35)])
```

Comment:
You can extract a constant defaultPadding = 20, because I see you use that value in different places.